direnvrc

```diff
@@ -6,7 +6,6 @@ NIX_DIRENV_VERSION=3.0.7
 # min required versions
 BASH_MIN_VERSION=4.4
```

Comment:
Probably we will have the same issue with the version comparison here at some point. We should investigate the behavior of sort.